### PR TITLE
Update docs to include instructions for unattended-upgrades under Ubuntu Focal

### PR DESCRIPTION
**Pull request by emkll**
## Status
Ready for review

## Description of Changes

* Towards https://github.com/freedomofpress/securedrop/pull/5684

## Testing
Instructions are logical and manual upgrade logic works in both Xenial (cron-apt) and Focal (unattended-upgrades)

## Release 
* The instructions are listed for both Xenial and Focal instances, though some 

## Checklist (Optional)

- [x] Doc linting (`make docs-lint`) passed locally
- [x] Doc link linting (`make docs-linkcheck`) passed
- [x] You have previewed (`make docs`) docs at http://localhost:8000

### CODE REVIEW

File: docs/admin.rst [hunk 1]
```diff
@@ -413,23 +413,42 @@ for how to enable error logging for the *Source Interface*.
...
-
-.. code:: sh
-
-  sudo cron-apt -i -s
-
-Depending on the nature of the update (e.g., if the ``tor`` package is upgraded
-and you are using SSH-over-Tor), your SSH connection may be interrupted, and you
+SecureDrop will update and reboot once per day. However, once a SecureDrop
+update `is announced`_ , you can opt to fetch the update immediately. Depending
+on the nature of the update (e.g., if the ``tor`` package is upgraded and you are
```
Comment (rocodes) [inline docs/admin.rst:418]: (Nit) for clarity's sake, I would move the "Depending on the nature of the update, ..." sentence to the very end of these instructions (current line 451), optionally in a ``note``.   But I don't think it's worth doing unless other people request other changes as well.

**Review by rocodes — DISMISSED** [review]
LGTM based on visual review. I left one small comment but it's not a showstopper.  Thanks @emkll :) 

(Making a mental note: reviewing this PR made me re-read the "[Partitioning the disks](https://github.com/freedomofpress/securedrop-docs/pull/146/files#diff-46b31285fdfbbef30df03572f45cd2d075175c829c274ce5a7c357d2abb42068)" section of the install docs, and maybe we should simplify that section just to say "don't enable FDE." The way we currently word it, introduce FDE, and then strongly recommend not to use it is a bit confusing. If others agree, I'll open an issue. However, I think that's out of scope for this PR).

**Comment by emkll** [comment]
https://github.com/freedomofpress/securedrop/pull/5684 has been merged, rebased on latest `main`

**Review by eloquence — APPROVED** [review]
LGTM. Did not test on a Focal instance but matches documented behavior of the `unattended-upgrades` tool. Made two small tweaks - lower-cased the version IDs as in the `os-release` file, and moved the note per Ro's comment.

**Pull request merged**